packages/react-components/src/components/RichTextEditor/RichTextSendBox.tsx

```diff
@@ -435,6 +435,11 @@ export const RichTextSendBox = (props: RichTextSendBoxProps): JSX.Element => {
 
   // ignore attachments error (errored attachment will not be added, shouldn't disable SendButton)
   const hasBlockingErrorMessages = useMemo(() => {
+    /* @conditional-compile-remove(file-sharing-acs) */
+    if (!hasIncompleteAttachmentUploads(attachments)) {
```

Comment:
This can also be moved to a separate hook and in this case AttachmentUploadsPendingError will be updated for all cases (send button and blocking error)